database/factories/AirfieldFactory.php

```diff
@@ -10,5 +10,6 @@
         'icao' => (string) $faker->lexify('????'),
         'iata' => $faker->lexify('???'),
         'display_name' => $faker->city,
+        'airfield_group_id' => null,
```

Comment:
Shouldn't need this

Reply:
wanted to make it clear. no sense in leaving it